app/src/main/res/layout-land/profile_rename_activity.xml

```diff
@@ -58,9 +58,9 @@
           android:id="@+id/input_name"
           android:layout_width="match_parent"
           android:layout_height="wrap_content"
-          android:layout_marginStart="140dp"
-          android:layout_marginTop="4dp"
-          android:layout_marginEnd="140dp"
+          android:layout_marginStart="168dp"
+          android:layout_marginTop="32dp"
```

Comment:
![Screenshot_1588244640](https://user-images.githubusercontent.com/9396084/80703446-6e98f400-8b00-11ea-8afa-2dc811ff5fc1.png)

The Right side of button should be aligned with the ProfileInputView, you can compare this with https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/6f83fcd9-e72d-4941-a16f-6754a869e083/RG-MN-Register-Empty-37/

Also, while you are at it, just add paddingBottom `68dp` for `ConstraintLayout`, so that the SaveButton can look better.

Reply:
This pr includes changes related only ProfileInputView  for other issues we may create issues.

Reply:
https://github.com/oppia/oppia-android/issues/1040